### ISSUE TITLE
Add Table in Chrome does not work

### ISSUE THREAD

**Issue by thomasdilts** [ondras/wwwsqldesigner#285]
Add Table does not work in google chrome. It works in all the other browsers I have tried.

**Comment by ondras** [comment]
I think you will have to be more specific.

I just tried that with Chrome 75, Windows 10, https://ondras.zarovi.cz/sql/demo/ -- and it normally worked.

What exactly is your setup and what happens? Does the Console show something relevant?

**Comment by thomasdilts** [comment]
Wow. That is really strange. Now, when I try in chrome it works perfectly. It really is a wonderful tool. Thank you so much.

**Comment by micodel** [comment]
I appear to be having this issue. It was working off and on, but now just wont work. Ive tried restarting, new project, etc

**Comment by ondras** [comment]
Hi @micodel,
thanks for your detailed report. Can you please clarify (using words or screenshots) these two, please?

> the cursor changed into a gray click and drag box selector.

> stuck on the closing of the gray click and drag box

In particular: adding a table is a two-step process, where you first have to click the workspace in order to determine the table's position; then a popup box is opened where you fill in the table's name. When does the behavior described happens?

**Comment by micodel** [comment]
@ondras 

Step 1: Click the "Add Table" button. This changed the cursor to a cross.
Step 2: Click on the workspace to add the table.

Step 2 should normally drop the table box, but sometimes this second click fails to response, and you can just click and drag around a gray highlight window, which I seem unable to escape.

You can recreate this when the app is functioning by clicking and holding and dragging around the "Add Table" cursor.

I believe this is the same highlight box used to select multiple table boxes for rearrangement.

**Comment by micodel** [comment]
@ondras Also if this helps, I ran into the issue again today.

When I reloaded the browser tab, I received this alert.

![Screen Shot 2020-02-05 at 5 01 10 PM](https://user-images.githubusercontent.com/6325539/73891299-98ac8400-4839-11ea-874b-d08f9c5d3131.png)

**Comment by micodel** [comment]
@ondras P.S. Thanks for maintaining this app! Have used it for several years

**Comment by ondras** [comment]
> Step 2 is where the failure sometimes occurs. It should normally drop the table box, but sometimes this second click fails to respond, it gets stuck and you can only just click and drag around a gray highlight window, which I seem unable to escape.

If you *move* the mouse a bit between pressing and releasing the button, you will experience precisely the described behavior. That is because the sequence "press down and move" will initiate and create the multi-selection area; releasing the button subsequently will happen over this area and thus no "click" will be generated (mouse down/up must happen on the same element in order for "click" to be fired).

**Comment by nervouna** [comment]
I've encounter the same issue. However it worked fine in incontigo mode.

Chrome Version 83.0.4103.61 (Official Build) (64-bit)
Windows 10 18363.836

**Comment by rpgd60** [comment]
(re-reading the comments above,  I realize I am explaining --with different words-- what Ondras mentioned last February 12th

I encountered the same issue,  in Brave (1.7.92) and Firefox (77.0.1) running on Kubuntu 19.10
This does not look like a bug to me but simply a somewhat awkward unexpected GUI behavior 

- When clicking on "Add Table" the mouse pointer changes to a "cross hairs",  which many users associate with "click and drag" to select an area.   That was at least my immediate reaction.
- If you "click-and-drag" you start "selecting" an area in the drawing canvas -- see the pink rectangle in the figure.

![screenshot www sql designer](https://user-images.githubusercontent.com/5102312/86031842-ba034c00-ba36-11ea-93cf-ee506d6fda62.png)
- What one has to do instead is simply "Click" somewhere in the canvas.  Then the new table rectangle magically pops up.

In short, this is still a great application but in my opinion the GUI could be a bit more friendly :-)

**Comment by redvers** [comment]
Workaround:

If I click add table, nothing appears.  If I follow-up with clicking "aligntables" then a "newtable" object appears top-left which I can then click to edit / continue the workflow.

Hopefully this helps some people.

Thanks!
